sdk/appcenter/src/main/java/com/microsoft/appcenter/channel/DefaultChannel.java

```diff
@@ -601,17 +600,11 @@ private synchronized void handleSendingFailure(@NonNull GroupState groupState, @
         }
     }
 
-    /**
```

Comment:
Can you revise the doc instead of removing it?

Reply:
We never document `@Override` in the SDK.

Reply:
Oops. Didn't realize that's overridden method. 😄